core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowLocalRunnerPlugin.scala

```diff
@@ -48,11 +50,6 @@ object CloudflowLocalRunnerPlugin extends AutoPlugin {
   val Slf4jLog4jBridge = "org.slf4j" % "slf4j-log4j12" % "1.7.30"
   val Log4J            = "log4j" % "log4j" % "1.2.17"
 
-  // kafka config keys
-  val BootstrapServersKey = "bootstrap.servers"
-  val EmbeddedKafkaKey    = "embedded-kafka"
-
-  // kafka local values
   val KafkaPort = 9093
```

Comment:
Is this still used?

Reply:
here:
https://github.com/lightbend/cloudflow/pull/880/files/5ed09d3b5b9abcbac8285595d9cd06cbf4515650#diff-01d3a7c1e14d9d9f5cc6d61e91b7b89ea86087750afcf1f8690295de8e7efc2dR210